content/spaces/avatar.textile

```diff
@@ -0,0 +1,213 @@
+---
+title: Avatar stack
+meta_description: "Avatar stacks display the online status of members in a space."
+languages:
+  - javascript
+---
+
+Avatar stacks are the most common way of showing the status of members in an application by displaying an avatar for each. This will include whether they are online, or have recently disconnected. Additional information can also be provided, such as a profile picture and email address.
+
+An avatar stack is a reflection of who is present in a space for members to see who they are collaborating with. Events are emitted whenever a member enters or leaves a space, or updates their profile data. Whilst member events are published at the space-level, such as by entering a space, or updating a member's profile data, they can be specifically subscribed to at the member-level to keep an avatar stack updated in realtime.
```

Comment:
Not sure if we should say space-level and member-level. We should probably refer to the namespaces/ objects instead?